src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        return Color.values()[new Random().nextInt(Color.values().length)].toString();
```

Comment:
inlining operations is not always useful
Here you are creating a new `Random` object every time you are calling this method, let's avoid that by putting this `Random` to a class-level variable
Also worth to mention that it is recommended to call `name()` method for getting the `String` format of that enum instead of `toString()` try to think or google why

Reply:
fixed. Please check!